wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Subsystem.java

```diff
@@ -146,6 +146,18 @@ default Command runEnd(Runnable run, Runnable end) {
     return Commands.runEnd(run, end, this);
   }
 
+  /**
+   * Constructs a command that runs an action once and then runs another action every iteration
+   * until interrupted. Requires this subsystem.
+   *
+   * @param start the action to run on start
+   * @param run the action to run every iteration
+   * @return the command
+   */
+  default Command startRun(Runnable start, Runnable run) {
```

Comment:
What does default do here?

Reply:
`default` is used to define interface methods with a default implementation.